src/PROPOSAL/PROPOSAL/crosssection/parametrization/PhotoRealPhotonAssumption.h

```diff
@@ -32,16 +32,14 @@
 #include <memory>
 #include <unordered_map>
 
-#define PHOTO_PARAM_REAL_DEC(param, parent)                                    \
-    class Photo##param : public Photo##parent {                                \
+#define PHOTO_PARAM_REAL_DEC(param)                                            \
+    class Photo##param : public PhotoRealPhotonAssumption {                    \
```

Comment:
Since the photonuclear cross section (i.e. a charged particle interacting inelastically with a nucleus via a virtual photon) is now better decoupled from the photohadronic interaction (i.e. a real photon interacting with a nucleus, sometimes also called photoproduction), the parametrization to calculate the photonuclear interaction out of the photohadronic interaction in the GVD model (general vector dominance model) can also be modularized. Currently there is just one parametrization (BezrukovBugaev) but there are further parametrizations, like the BorogPetrukhin cross section (used in many applications e.g. in GEANT4), which can then be implemented.

Reply:
Then, there is no PhotoZeus or Rhode parametrization, but a PhotoBezrukovBugaev parametrization getting a PhotoproductionZeus, oder PhotoproductionBezrukovBugaev.
Like the ALLM Parametrizations get a shadowing parametrization.